### PR TITLE
2nd conversation, avatar transitions, and changes to how items work

### PR DESCRIPTION
**Pull request by thequeenofspades**
The "welcome party" conversation has been added. Conversations now have an `onEnd` function that triggers when the conversation ends. This is done through the use of `CustomActions.js` (in the `utils` folder) and is a tiny bit hacky.

The first conversation (`prologue01`) now transitions smoothly through a fade-to-black and fade-from-black into the second conversation (`prologue02`).

Avatars fade in and out when different speakers join the conversation. The currently shown avatar is the one that is speaking.

Each "inventory" (a.k.a. memory bank) item has its own entry, including description, in `items.json`. The `Clickables` object class works with `items.json` now.

Image assets have been organized into folders for avatars, items, and backgrounds.

image

### CODE REVIEW

File: src/objects/ConversationManager.js [hunk 1]
```diff
@@ -116,12 +118,12 @@ ConversationManager.prototype.takeActions = function() {
...
 
   for (var action in this.conversation[this.idx]['actions']) {
-    takeAction(this._game, action, this.conversation[this.idx]['actions'][action]);
+    this.takeAction(this._game, action, this.conversation[this.idx]['actions'][action]);
     return;
   }
 };
 
-function takeAction(game, action, value) {
+ConversationManager.prototype.takeAction = function(game, action, value) {
```
Comment (drcaramelsyrup) [inline src/objects/ConversationManager.js:126]: Awesome! Yeah I realized I forgot to mention the OOP style before.

File: src/utils/CustomActions.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  	this.fadeInAndOut(parseInt(action.substring('fadeInAndOut'.length)));
+  }
+  else if (action === 'startConv:prologue02') {
+    this._game.camera.fade('#000000', 2000);
+    this._game.camera.onFadeComplete.addOnce(function() {
+      this._game.camera.flash('#000000', 2000);
+      this.startConversation('prologue02');
+    }, this);
+  }
+};
```
Comment (drcaramelsyrup) [inline src/utils/CustomActions.js:31]: Is the intention for these custom actions to be very specific to the particular conversation, or might this eventually hold, like, general screenshaking events to be used as needed for any dialogue tree?

**Review by drcaramelsyrup — APPROVED** [review]
LGTM with a few comments! Peruse at your convenience.

File: src/utils/CustomActions.js (hunk 2 above)
Comment (thequeenofspades) [reply src/utils/CustomActions.js:31]: Eventually these should be pretty generalizable? I'm still trying to figure out a compact way to store this kind of information in a conversation's JSON file.

Comment (drcaramelsyrup) [reply src/utils/CustomActions.js:31]: Got it. We can talk about it on Saturday as well–I imagine there should be a set of functions which are fairly generalizable. I also just needed to get these changes merged in for my own progress.

**Pull request merged**